django/db/backends/utils.py

```diff
@@ -44,12 +45,20 @@ def __exit__(self, type, value, traceback):
 
     # The following methods cannot be implemented in __getattr__, because the
     # code must run when the method is invoked, not just when it is accessed.
+    # Keyword parameters for callproc are not supported regarding PEP 249, but
+    # underlying backend may supports keyword parameters (e.g cx_Oracle).
+    def callproc(self, procname, params=None, kparams=None):
+        if kparams is not None and not self.db.features.supports_callproc_kwargs:
+            raise NotSupportedError(
+                'Keyword parameters for callproc are not supported on this '
+                'database backend.'
+            )
 
-    def callproc(self, procname, params=None):
         self.db.validate_no_broken_transaction()
         with self.db.wrap_database_errors:
-            if params is None:
-                return self.cursor.callproc(procname)
+            params = params or ()
```

Comment:
The current idiom might be required because some backends (perhaps third-party ones) choke on empty `params`. I'd keep it.